monkey/monkey_island/cc/services/config_schema/ransomware.py

```diff
@@ -47,7 +47,14 @@
                             "files that you will allow Infection Monkey to encrypt. If no "
                             "directory is specified, no files will be encrypted.",
                         },
+
                     },
+
+                },
+                "readme_note": {
+                        "title": "",
+                        "type": "object",
+                        "description": "A README.txt file will be left alongside the encrypted files.",
```

Comment:
Can you add a screenshot that shows how this change renders?

Reply:
![image](https://user-images.githubusercontent.com/15820737/125068171-79acc400-e0b5-11eb-9d2f-bcb73497e16f.png)